packages/api-v4/src/databases/types.ts

```diff
@@ -174,7 +174,7 @@ interface BaseDatabase extends DatabaseInstance {
    * used_disk_size_gb is feature flagged by the API.
    * It may not be defined.
    */
-  used_disk_size_gb?: number;
+  used_disk_size_gb?: number | null;
```

Comment:
Does the API always return `used_disk_size_gb` in the response? If so, we can remove the optionality 

 ```suggestion
   used_disk_size_gb: number | null;
```

Reply:
@bnussman-akamai Asked backend team about this and they confirmed that `used_disk_size_gb` and `total_disk_size_gb` are always expected to be returned now. So I've updated both types so they're no longer tagged as optional and updated the change set comment.

Reply:
Thanks!